### DISCUSSION TITLE
rust

### DISCUSSION THREAD

**Discussion by jiuxinhetao** [GameServerManagers/LinuxGSM#3466] [General]
![image](https://user-images.githubusercontent.com/84342699/118530910-ca6c0480-b777-11eb-809e-7e32fdd443b3.png)
How can i solve this problem

**Comment by Capri205** [comment]
You are running Linux, so it will not load Windows dll libraries. I have those messages in my rust server logs and they are nothing to worry about.

**Comment by jialibee** [comment]
兄弟 你启动rustserver的时候有报sqlineexception吗。我的启动不起来。你的能正常启动吗。
[discussions https://github.com/GameServerManagers/LinuxGSM/discussions/3989](https://github.com/GameServerManagers/LinuxGSM/discussions/3989)
